types/node_information.go

```diff
@@ -93,7 +93,7 @@ func LoadNodeInformation(ctx context.Context, storage nodeenrollment.Storage, id
 		Id:    id,
 		State: opts.WithState,
 	}
-	if err := storage.Load(ctx, nodeInfo); err != nil {
+	if err := storage.Load(ctx, nodeInfo, opt...); err != nil {
```

Comment:
Should the comment above this function be updated? Seems it supports at least the new option (by passing it into `storage.Load`).